### PR TITLE
Write lab to practice if conditionals

### PR DESCRIPTION
**Pull request by kcleland0818**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — COMMENTED** [review]
The rolling a die example doesn't need a conditional since they could just
use string concatenation to achieve the output.   Maybe have them print random
card suit and face value(2 of hearts, 5 of diamonds), weather forecast based
on random temperature, lunch special of the day, etc.  Require them to run the program enough time to confirm each branch executed?

**Comment by kcleland0818** [comment]
> The rolling a die example doesn't need a conditional since they could just use string concatenation to achieve the output. Maybe have them print random card suit and face value(2 of hearts, 5 of diamonds), weather forecast based on random temperature, lunch special of the day, etc. Require them to run the program enough time to confirm each branch executed?

Yeah, I realize that. I did specify in the instructions that they **must** use a conditional and figured I could apply some rules to check for an if/else if/else block. Let me think of another random game.

File: README.md [hunk 1]
```diff
@@ -2,30 +2,188 @@
...
+instructions below to see where your code should go.
 
 ## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+There are three parts to this lab:
+
+- Simulate flipping a coin.
+- Simulate rolling a die.
```
Comment (developerjaym) [inline README.md]: I think this needs updated to rock paper scissors.

File: README.md [hunk 2]
```diff
@@ -2,30 +2,188 @@
...
+and scissors as the integer `2`.
+
+- Use the `nextInt()` method to generate a pseudorandom `int` value from 0 - 2
+  inclusive to represent two opponents playing the game.
+- Use a chained conditional statement to determine what each opponent plays.
+  - You _must_ tell the user what each opponent plays.
+    - If opponent 1 plays rock, print "Opponent 1 plays rock."
+    - If opponent 1 plays paper, print "Opponent 1 plays paper."
+    - If opponent 1 plays scissors, print "Opponent 1 plays scissors."
+  - You _must_ tell the user which component won or if it was a tie.
```
Comment (developerjaym) [inline README.md]: 'Component' should be 'Opponent'

**Comment by developerjaym** [comment]
> > The rolling a die example doesn't need a conditional since they could just use string concatenation to achieve the output. Maybe have them print random card suit and face value(2 of hearts, 5 of diamonds), weather forecast based on random temperature, lunch special of the day, etc. Require them to run the program enough time to confirm each branch executed?
> 
> Yeah, I realize that. I did specify in the instructions that they **must** use a conditional and figured I could apply some rules to check for an if/else if/else block. Let me think of another random game.

Rock, Paper, Scissors should be a fun challenge for them. Great idea!

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md]: Yes! Thanks for catching this!

File: README.md (hunk 2 above)
Comment (kcleland0818) [reply README.md]: Yes! Typo!

**Review by linda-seiter — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Pull request merged**